Car_owners/docker-compose.yml

```diff
@@ -5,12 +5,10 @@ services:
     build:
       context: .
       dockerfile: Dockerfile_postgres
+    env_file:
+      - .env
     volumes:
       - ./data/db:/var/lib/postgresql/data
-    environment:
```

Comment:
actually I would leave that, it's pretty common to have such default values set here